ring/src/lib.rs

```diff
@@ -26,51 +24,6 @@ pub type RingProof<F, CS> = Proof<F, CS, RingCommitments<F, <CS as PCS<F>>::C>,
 /// Polynomial Commitment Schemes.
 pub use fflonk::pcs;
 
-// Calling the method for a prime-order curve results in an infinite loop.
-pub fn find_complement_point<Curve: SWCurveConfig>() -> Affine<Curve> {
```

Comment:
So as we talked for SW the seed should be out and for TE should be in so I don't think we could get rid of these just like that?

Reply:
right